include/openmc/mesh.h

```diff
@@ -271,7 +272,7 @@ class RectilinearMesh : public StructuredMesh
   // Constructors
   RectilinearMesh() = default;
   RectilinearMesh(pugi::xml_node node);
-
+  
```

Comment:
remove added whitespace please. Did you apply clang-format? May want to do that.